packages/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
```

Comment:
You shouldn't need any of the ignores now

Reply:
Thanks for the note. I have removed all the import  error ignore comments, except for the following import in file SplitButton.stories.tsx which threw the error "Cannot find module '@fluentui/react-menubutton' or its corresponding type declarations.":

    import { MenuButtonProps } from '@fluentui/react-menubutton';

Any ideas how to fix the import?